src/FurnaceRecipe.cpp

```diff
@@ -171,7 +172,14 @@ void cFurnaceRecipe::AddRecipeFromLine(const AString & a_Line, unsigned int a_Li
 		return;
 	}
 
-	const AStringVector & InputSplit = StringSplit(Sides[0], "@");
+	const AStringVector & ItemDamageSplit = StringSplit(Sides[0], "^");
+	AString  ItemString = ItemDamageSplit[0];
+	if ((ItemDamageSplit.size() > 1) && (ItemDamageSplit[1] == "-1"))
+	{
+		IgnoreItemDamage = true;
+	}
```

Comment:
Just reuse the parsing code in ParseItem and check if `Recipe->In->m_ItemDamage < 0` later.

Reply:
@bearbin That won't work, the problem is a armor can be damaged and the damage is stored in that value

Reply:
That value isn't the damage of the in-game item, it's the damage of the recipe item so it should work. The damage of the item itself is in `a_Ingredient.m_ItemDamage` AFAIK.